gluster/driver/tools.go

```diff
@@ -11,18 +11,26 @@ import (
 )
 
 const (
-	validHostnameRegex = `(([a-zA-Z0-9]|[a-zA-Z0-9][a-zA-Z0-9\-]*[a-zA-Z0-9])\.)*([A-Za-z0-9]|[A-Za-z0-9][A-Za-z0-9\-]*[A-Za-z0-9])`
+	validVolUriRegex = `([^:]+?):\/?([^\/]+)(/.+)?`
```

Comment:
I am opting not to make the regex too strict, if the user decides to put in wrong input then it will just fail anyway.  So I simplified it here

Reply:
It is better to fail at creation of volume and not mount and be strict to validate that. 
It mostly important (at least for me) when using distributed docker like swarm because it will at creation of service and not try to mount (and remount and remount and ...) a false volume.
It allow for better error handling also otherwise it will be another case behind mount failling with just `exit status 1`. https://github.com/sapk/docker-volume-gluster/issues?utf8=%E2%9C%93&q=is%3Aissue+exit+status+1

I will fix this if you prefer for the rest than you very much.